aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -1162,9 +1163,75 @@ void vdot<c10::complex<double>>(CUDABLAS_DOT_ARGTYPES(c10::complex<double>)) {
                                    reinterpret_cast<cuDoubleComplex*>(result)));
 }
 
-// This guards blocks use of getrsBatched, geqrfBatched, getrfBatched, getriBatched on platforms other than cuda
-#ifdef CUDART_VERSION
 
+#ifdef USE_ROCM
+
+
+template <>
+void getrsBatched<float>(HIPBLAS_GETRS_ARGTYPES(float)) {
+  TORCH_HIPBLAS_CHECK(cublasSgetrsBatched(
```

Comment:
This might be in the "nit" category, but any reason you chose to keep the cublas API names even though you're explicitly ifdefing for ROCm? It seems odd to rely on hipification to switch the apis to hipblas when the rest of the surrounding code is hipblas-specific anyway? cc @jeffdaily

Reply:
That's a good point. I suppose my thinking was that once we move to hipblas proper, the ifdefs will go away. At that point, the plumbing for hipification will already be there. One less thing to change when we consolidate back to hipblas